.gitattributes

```diff
@@ -1,2 +1,6 @@
-*.go text eol=lf 
-*.txt text eol=lf
\ No newline at end of file
+**/*.go text eol=lf
+**/*.txt text eol=lf
+**/*.sh text eol=lf
```

Comment:
Does it makes sense to keep one `.gitattributes` file kept separate for the template repositories?

Reply:
Agreed - When the templates are generated it should include its own `.gitattributes` file.  We can either use repoman to include it or add any template specific files into the templates.

We should figure this out before moving this forward.